src/builtin/module_builtin_ast_adapters.cpp

```diff
@@ -1530,10 +1657,16 @@ namespace das {
                 return nullptr;
             }
         }
+        DECLARE_SERIALIZABLE ( TypeInfoMacro, TypeInfoMacroAdapter )
+        virtual void serialize ( AstSerializer & ser ) {
+            ser << _das_class_method_offset;
+            TypeInfoMacro::serialize(ser);
+        }
     protected:
         void *      classPtr;
         Context *   context;
     };
```

Comment:
Like I've mentioned earlier, no macros need to be serialized. When loaded, we create macro module, and in its turn it will create all necessary macros.